### DISCUSSION TITLE
How to map mimic cxr and mimic iv based on study_id?

### DISCUSSION THREAD

**Discussion by qingerVT** [MIT-LCP/mimic-code#1102] [MIMIC-General]
It's clear that we are able to map mimic cxr and mimic iv based on `patient_id`. Any way we can go a bit further to use`study_id`? For example, how to align `study_id` and `icustay_id,` `charttime`?

**Comment by briangow** [comment]
@ qingerVT I'm converting this to a discussion as we now prefer that issues are only used for problems with the code.

**Comment by briangow** [comment]
@qingerVT , I'm not sure if I fully understand what you're asking but you can use variables such as `subject_id` to merge information across tables.  For example, both the mimic_cxr.record_list table and the mimic_icu.icustays table contain `subject_id`.  Therefore you can do a [JOIN](https://en.wikipedia.org/wiki/Join_(SQL)) on `subject_id` to get all of the ICU stay information alongside the CXR information.  If this doesn't help please provide more details regarding what you're trying to do.

**Reply by qingerVT** [reply]
I understand how to merge information via `subject_id`. My quesiton is how to match `study_id`? A patient might have multiple studies.

**Reply by briangow** [reply]
What do you want to match `study_id` with?  On the [MIMIC website](https://mimic.mit.edu/docs/iv/modules/cxr/record_list/#study_id) you can see that for MIMIC-CXR `study_id` is a unique identifier for the radiology report written for the given chest x-ray.  Therefore, you can use it to JOIN information across MIMIC-CXR tables which have a `study_id` column but you can't JOIN on this key with any other tables.
